### DISCUSSION TITLE
Unable to execute 'Getting Started' examples

### DISCUSSION THREAD

**Discussion by omarmao12** [theOehrly/Fast-F1#208] [Q&A]
Hi, 

I have looked online and am unable to find a solution here. I have pip installed fastf1 and am running python 3.9 but for some reason I get error messages for even the basic examples in the 'Getting Started' guide. 

I can't even load a session, I keep getting this error: 'Session' object has no attribute 'load'. Am I missing something very obvious here? 

Thanks in advance!

**Comment by theOehrly** [comment]
Can you please post the full output of the script including logged messages and error traceback.

**Reply by omarmao12** [reply]
Hi theOehrly, 

Thanks for responding so quickly. It has now been resolved. I changed the Python version on Visual Studio from 3.9.9 to 3.9.7 and now the scripts run fine. 

I have attached the error message anyway in case someone else has this issue.

<img width="743" alt="Screenshot 2022-07-11 at 19 23 24" src="https://user-images.githubusercontent.com/25973132/178332080-577a4c5a-7c93-4add-b859-492882be7c1d.png">

**Reply by theOehrly** [reply]
Great that you resolved it. But this is really weird. Any chance that you had an older version of FastF1 installed previously in Python 3.9.7? The version of python really shouldn't make a difference here. But the `load` function was only introduced in FastF1 v2.2 I think. So with older versions you'd get that error.

**Reply by omarmao12** [reply]
I did have an older version of FastF1 installed in Python 3.9.7 but that shouldn't stop working once I use 3.9.9 right? Regardless, I will let you know if anything else that is weird pops up in the meantime. Thanks for your support!

**Reply by theOehrly** [reply]
Well no, it shouldn't. There should be no difference.
